mars/optimizes/ne.py

```diff
@@ -66,83 +65,17 @@
     reduction.TensorProd,
     reduction.TensorMax,
     reduction.TensorMin
-}
-
-
-def _check_reduction_axis(node):
-    return len(node.op.axis) == 1 or len(node.op.axis) == node.ndim
-
-
-def _support(node):
-    op_type = type(node.op)
-    if op_type in REDUCTION_OP:
-        return _check_reduction_axis(node)
-    return op_type in SUPPORT_OP
-
-
-def _transfer_op(node):
-    op = node.op
-    if type(op) in REDUCTION_OP and not _check_reduction_axis(node):
-        return op
-    return op
+)
```

Comment:
Why use `tuple` ? The time complexity of `in` is O(n) for tuple while O(1) for set.

Reply:
isinstance(xx,classinfo). classinfo only support tuple